### PR TITLE
[datadog_sensitive_data_scanner_rule] Add priority level in rules

### PR DESCRIPTION
**Pull request by trosenblatt**
[JIRA_TICKET](https://datadoghq.atlassian.net/browse/SDS-219) This enable support for priority field in SDS Rules in datadog terraform provider

### CODE REVIEW

**Review by vinckama — DISMISSED** [review]
LGTM

File: docs/resources/sensitive_data_scanner_rule.md [hunk 1]
```diff
@@ -81,6 +82,7 @@ resource "datadog_sensitive_data_scanner_rule" "mylibraryrule" {
 - `name` (String) Name of the rule.
 - `namespaces` (List of String) Attributes included in the scan. If namespaces is empty or missing, all attributes except excluded_namespaces are scanned. If both are missing the whole event is scanned.
 - `pattern` (String) Not included if there is a relationship to a standard pattern.
+- `priority` (Number) Priority level of the rule (optional). Used to order sensitive data discovered in the sds summary page. It must be between 1 and 5 (1 is the most important).
```
Comment (rtrieu) [inline docs/resources/sensitive_data_scanner_rule.md:85]:
```suggestion
- `priority` (Number) Priority level of the rule (optional). Used to order sensitive data discovered in the SDS summary page. It must be between 1 and 5 (1 being the most important).
```

**Review by rtrieu — APPROVED** [review]
Minor suggestion, but otherwise looks good.

**Review by vinckama — DISMISSED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,9 @@
+## 3.39.0 (April 22, 2024)
+
+### IMPROVEMENTS
+
+* [datadog_sensitive_data_scanner_rule] Add priority level in the rule in https://github.com/DataDog/terraform-provider-datadog/pull/2376
+
```
Comment (skarimo) [inline CHANGELOG.md:6]:
```suggestion
```
This is autogenerated during release process

**Review by skarimo — DISMISSED** [review]

File: docs/resources/sensitive_data_scanner_rule.md (hunk 1 above)
Comment (rtrieu) [inline docs/resources/sensitive_data_scanner_rule.md:85]:
```suggestion
- `priority` (Number) Priority level of the rule (optional). Used to order sensitive data discovered in the SDS summary page. It must be between 1 and 5 (1 being the most important).
```

**Review by rtrieu — DISMISSED** [review]
Same feedback.

**Comment by trosenblatt** [comment]
Sorry @rtrieu I accepted your changed but the script to generated docs overrides it. I think it is good know

**Comment by rtrieu** [comment]
Hi @trosenblatt I think it did it again 😅

**Comment by trosenblatt** [comment]
@rtrieu what do you mean it did it again ? I replaced the is with being. Is there anything else ?

**Comment by rtrieu** [comment]
Hi @trosenblatt I meant that it looks like the script overrode my suggestion and review. I will approve again now!

**Review by rtrieu — APPROVED** [review]
LGTM!

**Review by skarimo — APPROVED** [review]

**Pull request merged**